android-sdk/src/main/java/com/mobilecoin/lib/UnsignedLong.java

```diff
@@ -2,13 +2,15 @@
 
 package com.mobilecoin.lib;
 
+import android.os.Parcel;
+import android.os.Parcelable;
+
 import androidx.annotation.NonNull;
 
-import java.io.Serializable;
 import java.math.BigInteger;
 import java.util.Objects;
 
-public final class UnsignedLong extends Number implements Comparable<UnsignedLong>, Serializable {
+public final class UnsignedLong extends Number implements Comparable<UnsignedLong>, Parcelable {
```

Comment:
I removed the explicit Serializable implementation here because, by extending Number, UnsignedLong already implicitly implements Serializable